js/packages/web/src/actions/getStorefront.ts

```diff
@@ -83,17 +83,17 @@ const fetchFromSource = async (
     }, {});
     let pubkeydenyListCacheKey: Array<String> = [];
     try {
-      const cachedValue = await redisClient.get(denyListCacheKey);
+      const cachedValue = (await redisClient.get(denyListCacheKey)) || '{}';
       pubkeydenyListCacheKey = JSON.parse(cachedValue) || [];
 
       if (!pubkeydenyListCacheKey.length) {
         pubkeydenyListCacheKey = await fetchdenyListCacheKey();
-        // @ts-ignore
         redisClient.set(
           denyListCacheKey,
+          // @ts-ignore
           JSON.stringify(pubkeydenyListCacheKey.result),
         );
-        redisClient.expire(denyListCacheKey, 3600); // cache the entire list for an hour
+        redisClient.expire(denyListCacheKey, 1800); // cache the entire list for an hour
```

Comment:
I see that the commit says five minutes, but this number is 30 minutes and the comment says one hour lol